app/models/concerns/user_multifactor_methods.rb

```diff
@@ -2,38 +2,14 @@ module UserMultifactorMethods
   extend ActiveSupport::Concern
 
   included do
+    include UserOtpMethods
```

Comment:
I think it may make more sense for this to be included in `User.rb`. `UserWebauthnMethods` lives in there.

Reply:
Nevermind, this was addressed in #3808